books/md/Core-Rulebook.md

```diff
@@ -123,28 +150,29 @@ These categories are broken down by “roll range”. In order to determine roll
 
 To do this, calculate the minimum score you can get in your attempt (`STAT + SKILL + # SKILLS`), (where `# SKILLS` is the number of supporting skills you’re using). Then subtract this number from the `DC` (`DC-MIN`). This number is your roll range. An example? Sure:
 
-> You: “I’m going to try and pick the lock”
+> You: “I’m going to try and pick the lock.”
 >
-> > *You then roll a `d20` (`15`), and add your DEXT (`0`) and Lockpicking (`3`).*
+> > *You then roll a `d20` (`15`), and add your DEXT (`0`) and Lockpicking (`3`). Finally, you add one extra point because you're applying your supporting skill of Locksmithing.*
 >
-> > *Your total is `18`.*
+> > *Your total is `19`.*
 >
 >
 > GM: “You succeed, the lock clicks open revealing...”
 >
 > > *The DC was `16`. Your minimum score was `DEXT + Lockpicking` (`3`).*
 >
-> > *`16 - 4 = 13`. `12 < 13 ≤ 20`. This lock was of hard difficulty.*
+> > *`16 - 3 = 13`. `12 < 13 ≤ 20`. This lock was of hard difficulty.*
```

Comment:
Which would make it

> `16 - 4 = 12`. `1 < 12 ≤ 12`. This lock was of moderate difficulty.

But I might consider changing the DC to, say, 13, to make the math more clear:

> `13 - 4 = 9`. `1 < 9 ≤ 12`.